From b7167060af7e1e8c73f2075979fa1032526a6154 Mon Sep 17 00:00:00 2001
From: ChenXiaoSong <chenxiaosong@kylinos.cn>
Date: Sun, 7 Dec 2025 12:44:16 +0800
Subject: [PATCH 13/30] smb/client: remove useless elements from
 ntstatus_to_dos_map array

The last element in the array is no longer needed.

Signed-off-by: ChenXiaoSong <chenxiaosong@kylinos.cn>
---
 fs/smb/client/netmisc.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/fs/smb/client/netmisc.c b/fs/smb/client/netmisc.c
index 55100c2c14cf..4e6312593d6f 100644
--- a/fs/smb/client/netmisc.c
+++ b/fs/smb/client/netmisc.c
@@ -793,8 +793,7 @@ static struct ntstatus_to_dos ntstatus_to_dos_map[] = {
 	ERRDOS, ERRnoaccess, 0xc0000290}, {
 	ERRDOS, ERRbadfunc, 0xc000029c}, {
 	ERRDOS, ERRsymlink, NT_STATUS_STOPPED_ON_SYMLINK}, {
-	ERRDOS, ERRinvlevel, 0x007c0001}, {
-	0, 0, 0 }
+	ERRDOS, ERRinvlevel, 0x007c0001},
 };
 
 static unsigned int ntstatus_to_dos_num = sizeof(ntstatus_to_dos_map) /
-- 
2.43.0

